RecoVertex/PrimaryVertexProducer_Alpaka/plugins/alpaka/PrimaryVertexProducer_Alpaka.cc

```diff
@@ -0,0 +1,166 @@
+#include "DataFormats/PortableVertex/interface/alpaka/VertexDeviceCollection.h"
+#include "DataFormats/PortableVertex/interface/VertexHostCollection.h"
+#include "DataFormats/BeamSpot/interface/BeamSpotHost.h"
```

Comment:
I believe this should be
```suggestion
#include "DataFormats/BeamSpot/interface/alpaka/BeamSpotDevice.h"
```